test/ecpair/index.js

```diff
@@ -234,12 +231,12 @@ describe('ECPair', function () {
     })
 
     describe('verify', function () {
```

Comment:
Might as well comment out the entire test vs having the shell run as if it's doing something